source/_footer.slim

```diff
@@ -2,7 +2,7 @@ footer
     .container
         .contact.mb-2.d-flex.justify-content-between
             .left
-                span &copy;2018-2019 iina.io
+                span &copy;2018-2021 iina.io
```

Comment:
```suggestion
                span &copy;2018-2022 iina.io
```

Reply:
why hardcode ?

Reply:
Yes, let's go with https://github.com/iina/iina-website/pull/13 as you (@vinayakkulkarni) commented earlier.